blocks/results-list-block/features/results-list/results/index.jsx

```diff
@@ -0,0 +1,347 @@
+              title={headlineText}
+              aria-hidden="true"
+              tabIndex="-1"
+            >
+              <Image
+                {...imageProperties}
+                url={imageURL !== null ? imageURL : targetFallbackImage}
+                alt={imageURL !== null ? headlineText : imageProperties.primaryLogoAlt}
+                resizedImageOptions={imageURL !== null
+                  ? extractResizedParams(element)
+                  : placeholderResizedImageOptions}
+              />
+            </a>
+          </div>
+        )
+        : null}
+      {(promoElements.showHeadline)
+        ? (
+          <div className="results-list--headline-container mobile-order-1">
+            <a href={url} title={headlineText}>
+              <Heading className="headline-text">{headlineText}</Heading>
+            </a>
+          </div>
+        )
+        : null }
+      {(promoElements.showDescription || promoElements.showDate || promoElements.showByline)
+        ? (
+          <div className="results-list--description-author-container mobile-order-3">
+            {(promoElements.showDescription && descriptionText)
+              ? (
+                <a href={url} title={headlineText}>
+                  <SecondaryFont as="p" className="description-text">
+                    {descriptionText}
+                  </SecondaryFont>
+                </a>
+              )
+              : null}
+            {(promoElements.showDate || promoElements.showByline)
+              ? (
+                <div className="results-list--author-date">
+                  {(promoElements.showByline)
+                    ? (
+                      <Byline
+                        content={element}
+                        list
+                        separator={promoElements.showDate}
+                        font="Primary"
+                      />
+                    )
+                    : null }
+                  {(promoElements.showDate)
+                    ? <ArticleDate classNames="story-date" date={displayDate} />
+                    : null }
+                </div>
+              )
+              : null}
+          </div>
+        )
+        : null }
+    </div>
+  );
+}));
+
+const Results = ({
+  customFields,
+  fusionContext,
+  fusionProperties,
+  imageProperties,
+}) => {
+  const {
+    lazyLoad,
+    listContentConfig: {
+      contentService,
+      contentConfigValues,
+    },
+  } = customFields;
+
+  const {
+    arcSite,
+    contextPath,
+    deployment,
+    isAdmin,
+  } = fusionContext;
+
+  const {
+    fallbackImage,
+    locale,
+  } = fusionProperties;
+
+  const [configuredOffset] = useState(
+    parseInt(contentConfigValues.offset, 10)
+    || parseInt(contentConfigValues.feedOffset, 10)
+    || parseInt(contentConfigValues.from, 10)
+    || 0,
+  );
+  const [configuredSize] = useState(
+    parseInt(contentConfigValues.size, 10)
+    || parseInt(contentConfigValues.feedSize, 10)
+    || 10,
+  );
```

Comment:
is there a reason why these are not set as variables without `useState`? They appear to help derive state elsewhere

```suggestion
  const configuredOffset = parseInt(contentConfigValues.offset, 10)
    || parseInt(contentConfigValues.feedOffset, 10)
    || parseInt(contentConfigValues.from, 10)
    || 0;

  const configuredSize = parseInt(contentConfigValues.size, 10)
    || parseInt(contentConfigValues.feedSize, 10)
    || 10;
```

Reply:
I'm removing those states... it was more of a knee jerk reaction to new items causing a re-render and I (wrongly) felt under the gun to get a working state quickly.